### PR TITLE
[AppService]: fix #17207 - for az webapp log config 'level' always defaults to verbose

### LINKED ISSUE #17207: webapp:az webapp log config level always defaults to verbose
When I enter this command:

az webapp log config -n myapp --resource-group myrg --application-logging filesystem --level **information**

I get this:

{
  "applicationLogs": {
    "azureBlobStorage": {
      "level": "Off",
      "retentionInDays": 3,
      "sasUrl": null
    },
    "azureTableStorage": {
      "level": "Off",
      "sasUrl": null
    },
    "fileSystem": {
      "level": "**Verbose**"
    }
  },
[see screenshot](https://imgur.com/a/nPtjiJa)

How can I set the level to a different value for application logging (filesystem)? No matter what, it is always defaulting to Verbose

By using the UI, we can set the level for "Application Logging (Filesystem)" and "Application Logging (Blob)" independently, but the cli, has just one parameter fot both, maybe it needs to be update to reflect this new feature.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bee2e287-5fb7-a849-9d3b-87013b783f16
* Version Independent ID: 97f9e1c4-1573-b776-0fce-53c65839d713
* Content: [az webapp log](https://docs.microsoft.com/en-us/cli/azure/webapp/log?view=azure-cli-latest#az_webapp_log_config)
* Content Source: [latest/docs-ref-autogen/webapp/log.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/latest/docs-ref-autogen/webapp/log.yml)
* Service: **app-service-web**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by Kotasudhakarreddy**
**Description**

#17207 
**Testing Guide**

`az webapp log config -n myapp --resource-group myrg --application-logging filesystem --level information`

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by qwordy** [comment]
Wait for Sisira's review

**Review by panchagnula — CHANGES_REQUESTED** [review]
Given --level is optional please verify running the command without --level doesn't cause exceptions. Ex. az webapp log config -n myapp --resource-group myrg --application-logging filesystem

File: src/azure-cli/azure/cli/command_modules/appservice/custom.py
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/custom.py:2093]: A code style issue. The name `level` indicates that it is an enum or string type. Possible values can be "INFO", "DEBUG", etc. In this line, it is a boolean type. Does it represent turn logging on or off?

Comment (Kotasudhakarreddy) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:2093]: > Just test with optional -level before merging

I have tested with optional `-level` it is not causing any exception. But it is not making any change to the log configuration. Current command will set it as `Verbose` by default. I did the same changes and tested in all the possible ways. Now it is working as expected.

Comment (Kotasudhakarreddy) [reply src/azure-cli/azure/cli/command_modules/appservice/custom.py:2093]: > A code style issue. The name `level` indicates that it is an enum or string type. Possible values can be "INFO", "DEBUG", etc. In this line, it looks like a boolean type. Does it represent turn logging on or off?

yes if user is not given the `-level` parameter level becomes `True` and it sets `Verbose`  level by default and when user want to `off` the logging then level becomes `false` and makes all the logging configurations `off`.

**Pull request merged**
